BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -121,6 +121,77 @@ extension Element {
         )
     }
 
+    /// Wraps the receiver in an accessibility element with the provided values.
+    ///
+    /// - Important: ⚠️: This will remove any previously resolved accessibility hint, identifier, or frame size of the element and all of its children. ⚠️
+    public func accessibilityElement(
+        label: String,
+        value: String,
+        traits: Set<AccessibilityElement.Trait>
+    ) -> AccessibilityElement {
```

Comment:
I'm fairly certain this (and likely a few others) will end up being ambiguous to the compiler which to chose. Could you add a test or something that references each to make sure the compiler doesn't get confused?